docs/getting-started/core-concept.md

```diff
@@ -37,11 +35,13 @@ Revision generates each time when the application deployed and holds all infos i
 
 Project is where you manage all the applications and collaborate with your team member. Project is one stand alone scope that separates it from other project.
 
-### Environment
+## Env
+
+Env refers to the environment for development, testing, and production and it can include multiple Targets. We define Environment in each Project and each Project can contain multiple Environments. Applications inside a same environment can visit and share resource with each other.
```

Comment:
```suggestion
Environment refers to the environment for development, testing, and production and it can include multiple Targets. We define Environment in each Project and each Project can contain multiple Environments. Applications in the same environment can visit and share resource with each other.
```